mail-client/alpine/alpine-2.21.1-r2.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools toolchain-funcs
+
+DESCRIPTION="An easy to use text-based based mail and news client"
+HOMEPAGE="https://www.washington.edu/alpine/ https://repo.or.cz/alpine.git/"
+GIT_COMMIT="843b2f16abfd949e09b1c5465387b1b0f724994a"
+MY_P="${PN}-${GIT_COMMIT::7}"
+SRC_URI="https://repo.or.cz/alpine.git/snapshot/${GIT_COMMIT}.tar.gz -> ${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ia64 ~ppc ~sparc ~x86"
+IUSE="doc ipv6 kerberos ldap libressl nls onlyalpine pam passfile smime spell ssl threads"
+
+DEPEND=">=sys-libs/ncurses-5.1:0=
+	kerberos? ( app-crypt/mit-krb5 )
+	ldap? ( net-nds/openldap )
+	pam? ( sys-libs/pam )
+	spell? ( app-text/aspell )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+"
+RDEPEND="${DEPEND}
+	app-misc/mime-types
+"
+
+HTML_DOCS=("doc/tech-notes/")
+
+PATCHES=(
+	"${FILESDIR}/${PV}-nopam.patch"
```

Comment:
The patchfile should be identifiable from name, ie, use `${P}-nopam.patch` as the filename rather than `${PV}`.

https://dev.gentoo.org/~vapier/clean-patches

Reply:
Fixed